openapi/task_execution_service.openapi.yaml

```diff
@@ -741,6 +764,19 @@ components:
           example: 2020-10-02T10:00:00-05:00
           readOnly: true
       description: Task describes an instance of a task.
+    tesCallbackStatus:
```

Comment:
Looks to me like the `schemas` are listed in alphabetical order, so the `tesCallbackStatus` should be inserted as the first schema, before `tesCancelTaskResponse` instead.